### ISSUE TITLE
Create a FAQ for the promotional side of ChRIS Store webpages

### ISSUE THREAD

**Issue by mairin** [FNNDSC/ChRIS_store_ui#213]
The FAQ should have its own item in the top navbar.

The FAQ should include: 
- how to install a plugin
- how ChRIS store works (needs github+dockerhub) (see #149)
- how to run your own ChRIS server

**Comment by Hina-softwareEngineer** [comment]
Hello, I am new to open-source and want to contribute here. Can I take this issue as a head-start?

**Comment by rudolphpienaar** [comment]
Sure!

**Comment by Hina-softwareEngineer** [comment]
Thanks, I will setup the chris store ui project and start working. I have already setup the chris core UI project.  Can you please assign this issue to me?

**Comment by rudolphpienaar** [comment]
Well, feel free to just start. We don't need at this stage to formally assign anything to anyone.

**Comment by rudolphpienaar** [comment]
Also, I expect you can't simply fully jump in on your own. Please reach out and/or describe what you are thinking of doing first -- otherwise you run the real risk of:

1. getting totally stuck
2. doing work that is not in line with this issue.

**Comment by Hina-softwareEngineer** [comment]
Yes, You are right. I am coming from the outreachy program. I want to be familiar with the repository and its code and contribute to it. Can you please help me with that?

**Comment by katoj65** [comment]
Hello, Am an outreachy applicant, I need help to contribute on this issue
